api.md

```diff
@@ -718,6 +718,9 @@ all attached hypertables.
 | ----------- | -----------            |
 | `node_name` | Name of the data node. |
 
+>:WARNING: Although the `cascade` parameter is not strictly a required argument,
+you *MUST* set it to `true` in this current release to avoid putting TimescaleDB in an error state.
```

Comment:
```suggestion
you *MUST* set it to `true` in this current release to avoid putting TimescaleDB in a recoverable error state.
```
as we discussed in Slack, so it is not so scary.

Reply:
Done!